src/chart/line/LineView.ts

```diff
@@ -609,6 +610,8 @@ class LineView extends ChartView {
             }
         }
         this._clipShapeForSymbol = clipShapeForSymbol;
+        const visualColor = getVisualGradient(data, coordSys)
```

Comment:
I think it's not appropriate to use a gradient on the endLabel. Perhaps it's better to pick the last color from the gradient stops?

Reply:
Yes, It should use `convertToColorString ` to convert it @pissang

Reply:
Nice fix! I didn't notice the exists of `convertToColorString`